client/deposits/details/index.tsx

```diff
@@ -7,7 +7,15 @@ import React from 'react';
 import { dateI18n } from '@wordpress/date';
 import { __, sprintf } from '@wordpress/i18n';
 import moment from 'moment';
-import { Card } from '@wordpress/components';
+import {
+	Card,
+	CardBody,
+	CardHeader,
+	ExternalLink,
+	// @ts-expect-error: Suppressing Module '"@wordpress/components"' has no exported member '__experimentalText'.
+	// eslint-disable-next-line @wordpress/no-unsafe-wp-apis -- used by TableCard component which we replicate here.
+	__experimentalText as Text,
```

Comment:
This isn't optimal, but I figured that using the same components as `@woocommerce/components` `TableCard` is preferred to re-implementing the styles with other components or HTML.

https://github.com/woocommerce/woocommerce/blob/42eea4a1db769c5203d2ba60b4e912919eb5051e/packages/js/components/src/table/index.tsx#L14-L15